components/ipfs/import/ipfs_import_worker_base.cc

```diff
@@ -174,7 +176,7 @@ void IpfsImportWorkerBase::OnImportAddComplete(
         simple_url_loader_->ResponseInfo()->headers->response_code();
 
   bool success = (error_code == net::OK && response_code == net::HTTP_OK);
-  if (success) {
+  if (success && response_body) {
```

Comment:
Should we change success state to false if response_body is null?

Reply:
changed `IpfsImportWorkerBase::ParseResponseBody` to return false in case of invalid pointer